.history/public/index_20230327085330.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/style.css">
+    <title>Ayomide S.E.P</title>
+</head>
+<body>
+    
+    <header class="header header-top">
+        <div class="header-inner sp-btw-flex">
+            <div class="heoader-log">
+                <img src="../assets/Logo Placeholder.png" alt="">
+            </div>
+            <div class="header-mobile-icon">
+                <img src="../assets/Normal Button/Tertiary/Enabled.png" alt="">
+            </div>
+        </div>
+    </header>
+
+    <section class="intro-section">
+        <h2 class="intro-title">I am Ayomide.
+            Glad to see you!</h2>
+            <p class="intro-paragraph">
+                I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                do not hesitate to contact me.
+            </p>
+            <button type="submit" class="let-connect-link">LET'S CONNECT</button>
+            <div class="social-icons">
```

Comment:
- [ ] This `let's connect` is only a simple text, not a button or a link, I would suggest you use an h2...h6 tag for it and remove the button.
- [ ] In adding to this comment, please make sure you stick to the original design by using the same color.
- [ ] see the font-weight used for your main title and compare it with the one in the figma on the left, I can see a small difference between the 2, kindly update that.
- [ ] and the way you have disposed of your `you` word, what do you think if you move a bit down to reassemble the figma one? 
- [ ] Comparing the 2 images, yours is missing the background image, I think adding it will make your design even better and clean referring to the original on the left
<img width="777" alt="1" src="https://user-images.githubusercontent.com/77843050/227919278-33483a59-0dc2-4254-9e92-021ac921fede.png">